doc/source/basics.rst

```diff
@@ -1788,6 +1788,46 @@ but occasionally has non-dates intermixed and you want to represent as missing.
 In addition, :meth:`~DataFrame.convert_objects` will attempt the *soft* conversion of any *object* dtypes, meaning that if all
```

Comment:
Why include deprecated methods in the doc?  Just rewrote the whole thing to not include `.convert_objects`.  Didn't really see a good way to include `.select_dtypes`, which is fine.